module/VuFind/src/VuFind/Controller/HoldsTrait.php

```diff
@@ -77,10 +75,15 @@ public function holdAction()
         if (!$gatheredDetails) {
             return $this->redirectToRecord();
         }
-
+        // the gatheredDetails['id'] is the original ID, but for API Holds (e.g. EDS)
+        // we may need to use the originalId. So only in that case we will set it to the
```

Comment:
Is this a typo? Should it say:

```suggestion
        // we may need to use the override ID. So only in that case we will set it to the
```